### ISSUE TITLE
Trackpad very slow on HP AM080NA Notebook

### ISSUE THREAD

**Issue by DaTrafficCone** [sebanc/brunch#1508]
Hi,
I recently installed brunch on my
HP Notebook and the trackpad has been acting up.
It is extremely slow and changing
the speed does nothing.

**Comment by KarlS0808** [comment]
You can apply the alternetive trackpad config by going into brunch config

**Comment by DaTrafficCone** [comment]
Thanks, ill try that

**Comment by DaTrafficCone** [comment]
It didnt work, i tried everything, i even used a HP chromebook recovery image.

**Comment by Mou786Aha** [comment]
> Hi, I recently installed brunch on my HP Notebook and the trackpad has been acting up. It is extremely slow and changing the speed does nothing.

Did you try sensitivity settings in settings? If not, find the setting related to trackpad settings and increase it.

**Comment by DaTrafficCone** [comment]
> > Hi, I recently installed brunch on my HP Notebook and the trackpad has been acting up. It is extremely slow and changing the speed does nothing.
> 
> Did you try sensitivity settings in settings? If not, find the setting related to trackpad settings and increase it.

Yes, i tried all of the fixes and the trackpad is still really slow and choppy.

**Comment by ewitsdevan** [comment]
I turned off "Enable tap-to-click" and it fixed my issue for me. Short and fast movements would register as taps.
